BE/types/index.ts

```diff
@@ -0,0 +1,79 @@
+export enum StatusState {
+  Pending = "Pending",
+  Success = "Success",
+  Failure = "Failure",
+}
+
+export enum ChallengeState {
+  InProgress = "InProgress",
+  Success = "Success",
+  Failure = "Failure",
+}
+
+export interface Task {
+  id: string;
+  description: string;
+}
+
+export interface Status {
+  state: StatusState;
+  updated: number;
```

Comment:
I suggest to use Date type instead of number because updated value is a timestamp